proposals/templates/extension-template.md

```diff
@@ -0,0 +1,116 @@
+<!-- {% raw %} -->
+
+# Feature name
+
+## Instructions
+
+> This template is a guideline for documenting a platform or vendor extension.
+> For a feature to be a conforming extension it must not change language
+> behaviors. It can introduce new builtin functions, data types, and annotating
+> attributes, but cannot introduce new language behaviors or rules.
+>
+> This template wraps at 80-columns. You don't need to match that wrapping, but
+> having some consistent column wrapping makes it easier to view diffs on
+> GitHub's review UI. Please wrap your lines to make it easier to review.
+>
+> Delete this Instructions section including the line below.
+
+---
+
+* Proposal: [NNNN](NNNN-filename.md)
```

Comment:
Any value in an EXT prefix or something to make them stand out from language proposals?

Reply:
My gut says for the proposals maybe not, but I'd like us to create a directory structure of extension specifications where we do something like what Khronos does where we have a platform/vendor prefix that also gets assigned.